### PR TITLE
Bump Cluster API to v1.1.3

### PR DESCRIPTION
**Pull request by saimanoj01**
### What this PR does / why we need it
This PR updates the version of ClusterAPI to v1.1.3.

### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note
Update Cluster API version to v1.1.3
```

### PR Checklist

- [ ] Squash the commits into one or a small number of logical commits
      
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

**Comment by saimanoj01** [comment]
//usebom:sandbox/v1.6.0-zshippable/398319358809498323/tkg-compatibility:v5

File: pkg/v1/providers/bootstrap-kubeadm/v1.1.3/bootstrap-components.yaml [hunk 1]
```diff
@@ -6278,7 +6278,6 @@ spec:
         - --leader-elect
         - --metrics-bind-addr=localhost:8080
         - --feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},KubeadmBootstrapFormatIgnition=${EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION:=false}
-        - --bootstrap-token-ttl=${CAPBK_BOOTSTRAP_TOKEN_TTL:=15m}
```
Comment (vuil) [inline pkg/v1/providers/bootstrap-kubeadm/v1.1.3/bootstrap-components.yaml:6281]: is this no longer relevant/applicable?

**Review by vuil — COMMENTED** [review]
thanks @saimanoj01 , lgtm, just a had a quick question

File: pkg/v1/providers/bootstrap-kubeadm/v1.1.3/bootstrap-components.yaml (hunk 1 above)
Comment (saimanoj01) [reply pkg/v1/providers/bootstrap-kubeadm/v1.1.3/bootstrap-components.yaml:6281]: Yes it is removed from the template and is no longer applicable.

**Review by vuil — APPROVED** [review]
lgtm, thanks!

**Pull request merged**